src/main/java/com/fundation/webservice/model/Metadata.java

```diff
@@ -90,13 +89,24 @@ public void writeJsonFile(File file) {
         return result;
     }
 
+    /**
+     * Safe initialization of the list structure that stores the command line string to be passed to the
+     * run() method.
+     */
     public void initCommandLine() {
-        commandLine.clear();
-        commandLine.add("cmd.exe");
-        commandLine.add("/c");
+        super.initCommandLine();
         commandLine.add(USER_DIR + TOOLS_DIR + EXIFTOOL_DIR + "exiftool.exe");
     }
 
+    /**
+     * This method is used to determine the name of the output file.
+     * The output file is going to be named after the original file, but the extension will be replaced according
+     * to the metadata format that the client asks for.
+     *
+     * @param file
+     *            The handle (provided by a File object) to the param file that is going to be read by exiftool.
```

Comment:
please transform in unique one line this comment

Reply:
@param description text now spans just one line.